CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+##How To Contribute
+The ability to contribute and collaborate are what make **git** so great. This bot is still very young.
+We want to keep it as easy as possible to contribute changes that get things working and the bot more useful.
+
+There are a few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things.
+
+
+##Getting Started - How Can I Contribute
+**Quick Overview**
+
+```
+* Make Sure you have a Github Account :)
+* Fork the repository on GitHub
+* Create a topic branch from where you want to base your work.
+	* This is usually the master branch.
```

Comment:
This should be a fork of the `develop` branch, master is likely the release branch most of the times, so won't be the most up to date of code currently being developed.